core/src/main/scala/org/apache/spark/util/Clock.scala

```diff
@@ -65,3 +75,18 @@ private[spark] class SystemClock extends Clock {
     -1
```

Comment:
nit: not your fault, but generally when code is not reachable I like to throw an exception instead of returning a dummy value.